repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/image-block.ftl

```diff
@@ -0,0 +1,17 @@
+<#assign hst=JspTaglibs["http://www.hippoecm.org/jsp/hst/core"] >
+
+<#macro imageBlock imageBlock>
+    <@hst.link var="imgLink" hippobean=imageBlock.image/>
+    <figure class="nhsuk-image" style="width: 100%">
```

Comment:
I think we do not need `style="width: 100%"` as it is already covered by `nhsuk-image`. Could we remove it ?